azurerm/internal/services/network/public_ip_resource_test.go

```diff
@@ -677,6 +679,8 @@ resource "azurerm_public_ip" "test" {
   resource_group_name = azurerm_resource_group.test.name
   allocation_method   = "Static"
   domain_name_label   = "acctest-%d"
+  sku                 = "Standard"
+  sku_tier            = "Regional"
```

Comment:
since this is ForceNew, this'll recreate the resource when it changes - as such it can't go in the Update test (since whilst it'll work provisioning wise, it'll recreate the resource when doing so) - instead could we add two new tests (removing this from both the Basic/Update tests) for Global/Regional SKU Tiers?